### PR TITLE
New adapter that solely handles splitting streaming fees.

### PR DESCRIPTION
**Pull request by bweick**
Stripped out DebtIssuanceModule related logic from [FeeSplitAdapter](https://github.com/SetProtocol/index-coop-smart-contracts/blob/master/contracts/adapters/FeeSplitAdapter.sol)

### CODE REVIEW

File: contracts/adapters/StreamingFeeSplitAdapter.sol [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+        if (methodologistTake > 0) {
+            invokeManagerTransfer(address(setToken), methodologist, methodologistTake);
+        }
+
+        emit FeesAccrued(operator, methodologist, operatorTake, methodologistTake);
+    }
+
+    /**
+     * ONLY OPERATOR: Updates streaming fee on StreamingFeeModule. NOTE: This will accrue streaming fees though not send to operator
```
Comment (asoong) [inline contracts/adapters/StreamingFeeSplitAdapter.sol:98]: Where does it accrue to then?

Comment (asoong) [reply contracts/adapters/StreamingFeeSplitAdapter.sol:98]: Alternative, why not have this call `accrueFeesAndDistribute` first

File: utils/deploys/deployAdapters.ts [hunk 2]
```diff
@@ -5,14 +5,16 @@ import {
...
   FeeSplitAdapter,
-  GovernanceAdapter
+  GovernanceAdapter,
+  StreamingFeeSplitAdapter
 } from "../contracts/index";
 
 import { ExchangeIssuance__factory } from "../../typechain/factories/ExchangeIssuance__factory";
 import { ExchangeIssuanceV2__factory } from "../../typechain/factories/ExchangeIssuanceV2__factory";
-import { FeeSplitAdapter__factory } from "../../typechain/factories/FeeSplitAdapter__factory";
+import { FeeSplitAdapter__factory  } from "../../typechain/factories/FeeSplitAdapter__factory";
```
Comment (asoong) [inline utils/deploys/deployAdapters.ts:14]: nit extra space

File: utils/deploys/deployAdapters.ts [hunk 3]
```diff
@@ -5,14 +5,16 @@ import {
...
 import { FlexibleLeverageStrategyAdapter__factory } from "../../typechain/factories/FlexibleLeverageStrategyAdapter__factory";
 import { GovernanceAdapter__factory } from "../../typechain/factories/GovernanceAdapter__factory";
+import { StreamingFeeSplitAdapter__factory  } from "../../typechain/factories/StreamingFeeSplitAdapter__factory";
```
Comment (asoong) [inline utils/deploys/deployAdapters.ts:17]: nit extra space

**Review by asoong — COMMENTED** [review]
Im already starting to get confused between these adapters and the module adapters. I dont think its too late to start calling these something else, something that ties it to the `StreamingFeeSplitManagerExtension`

File: contracts/adapters/StreamingFeeSplitAdapter.sol (hunk 1 above)
Comment (bweick) [reply contracts/adapters/StreamingFeeSplitAdapter.sol:98]: Fees are auto-accrued on the `StreamingFeeModule` when the fee is updated. So they'll accrue to the manager contract. If someone wants to get them they could call `accrueFeesAndDistribute` and they can be distributed.

**Review by asoong — APPROVED** [review]

**Pull request merged**